doc/pages/administrator/095-advanced-operations.md

```diff
@@ -4,6 +4,14 @@ This section provides guides to perform some operations that have to be perfomed
 could result in data loss or other type of irrecoverable damage. *Always be careful while performing
```

Comment:
Avoid repetition and be more graceful: "This section provides guidance on some delicate operations that must be performed manually as they could potentially result in data loss or other types of irrecoverable damage."